src/components/Toast/Toast.vue

```diff
@@ -82,11 +84,17 @@ export default class MToast extends Vue.with(Props) {
   mounted() {
     if (this.timeout) {
       this.setTimeout()
+      this.$el.addEventListener('mouseenter', this.clearTimeout)
+      this.$el.addEventListener('mouseleave', this.setTimeout)
     }
   }
 
   unmounted() {
-    this.clearTimeout()
+    if (this.timeout) {
+      this.clearTimeout()
+      this.$el.removeEventListener('mouseenter', this.clearTimeout)
+      this.$el.removeEventListener('mouseleave', this.setTimeout)
+    }
```

Comment:
this is probably fine but technically `timeout` is reactive and its value could change after its mounted. This assumes it'll have its value when its mounted and never change.

Reply:
any harm in just calling `removeEventListenter` even if it was never added?

Reply:
Nope, that won't hurt anything. 

That should handle changes to timeout going from > 0 to 0. Again, this component isn't designed to be used on its own so probably not an issue.